pennylane/_qubit_device.py

```diff
@@ -769,20 +773,24 @@ def statistics(
               ``shot_range=[35, 135]``, ``bin_size=100``.
         """
         if observables is not None:
-            if isinstance(observables, QuantumScript):
-                circuit = observables
-                measurements = circuit.measurements
-            else:
+            warnings.warn(
+                message="The ``observables`` argument in ``QubitDevice.statistics`` is deprecated. "
+                "Please use ``circuit`` instead.",
+                category=UserWarning,
+            )
+            circuit = observables
```

Comment:
What if there is code that uses both `observables` and `circuit`??

Reply:
```suggestion
            measurements = observables
```

Reply:
Currently it defaults to using `observables` and ignores what was passed to `circuit`. This preserves the current behaviour on the rc branch. Is that what we want?

Reply:
updated